src/concrete/ml/quantization/quantized_module.py

```diff
@@ -598,9 +599,13 @@ def compile(
                 error to a default value.
             verbose (bool): Indicate if compilation information should be printed
                 during compilation. Default to False.
+            statuses (Optional[Dict[str, str]]): same dict as Concrete Python encryption statuses
```

Comment:
I would maybe better describe this, I think users should be able to understand what this is without having the need to look at CP !

Reply:
like `Encryption statuses for all input values, either "clear" or "encrypted" `